src/app/footer/footer.component.scss

```diff
@@ -0,0 +1,13 @@
+@import '../../variables';
+
+.footer{
+    background-color: $background-pale;
+    margin: 0 auto;
+};
+
+.btn-facebook {color:#fff; background-color:#3b5998;}

```

Comment:
1. please extract to constants
2. move to icon component scope
3. use HSLA format instead